engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableMaterial.java

```diff
@@ -37,7 +72,7 @@ public EnableMaterial(String materialName) {
         this.materialName = materialName;
     }
 
-    public String getMaterialName() {
+    private String getMaterialName() {
```

Comment:
Because getMaterialName() was not being used outside the class. But yeah, should be left public since it's a getter. I'll revert it back to public, make the changes and push.